trashcli/put.py

```diff
@@ -47,26 +47,25 @@ def __init__(self, stdout, stderr, environ, trasher):
 
     def run(self, argv):
         program_name = os.path.basename(argv[0])
-        parser = get_option_parser(program_name, self.stdout, self.stderr)
-        try:
-            (options, args) = parser.parse_args(argv[1:])
-
-            if len(args) <= 0:
-                parser.error("Please specify the files to trash.")
-        except SystemExit as e:
-            return e.code
-        else:
-            logger = MyLogger(self.stderr, program_name, options.verbose)
-            reporter = TrashPutReporter(logger, self.environ)
-            result = self.trash_all(args,
-                                    options.trashdir,
-                                    logger,
-                                    options.mode,
-                                    reporter,
-                                    options.forced_volume,
-                                    program_name)
-
-            return reporter.exit_code(result)
+        parser = make_parser(program_name, self.stdout, self.stderr)
+        options = parser.parse_args(argv[1:])
+        if len(options.files) <= 0:
+            sys.stdout = sys.stderr
+            parser.print_usage()
+            print("Please specify the files to trash.")
```

Comment:
Add this parameter file=sys.stderr, please

Reply:
Fixed.